src/NewRelic.LogEnrichers.Log4Net/NewRelic.LogEnrichers.Log4Net.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.8" />
-    <PackageReference Include="NewRelic.Agent.Api" Version="8.20.262" />
+    <PackageReference Include="NewRelic.Agent.Api" Version="8.35.0" />
```

Comment:
Even though we are safe referencing v8.35. In order to avoid any potential dependency problem, it is better to reference the lowest version of the API that support nullable type which is v8.32.

Reply:
Done